converter/templates/index.html

```diff
@@ -33,6 +33,13 @@ <h1 id="text-button" class="submittext">Submit Text</h1>
 
     <div id = "text">
         <form method="POST" id="text_input">
+            <label for="textformat">Choose a format:</label>
+            <select id="textformat" form="text_input">
+                <option value=".musicxml">MusicXML</option>
```

Comment:
Note that you have deployed this to `musicxml2abc.musicalkits.me`, and once you extend to more formats you need to reconsider the URL for deployment. I meant to bring this up after we get our first step stable, but you have jumped ahead.

Reply:
yes, it is not hard to change the prefix to another name.  `*.musicalkits.me`, any name works